framework/src/executioners/EigenExecutionerBase.C

```diff
@@ -67,6 +67,8 @@ EigenExecutionerBase::EigenExecutionerBase(const InputParameters & parameters)
   // FIXME: currently we have to use old and older solution vectors for power iteration.
   //       We will need 'step' in the future.
   _problem.transient(true);
```

Comment:
Do we still need this line?

Reply:
I'm not sure. Will check - but I'm not too concerned about a system that we're phasing out.